### ISSUE TITLE
Ability to select a specific GIF before sending

### ISSUE THREAD

**Issue by ebxm** [FinndropStudios/GifRocket#18]
Hello, 

I would be interested to have a similar feature as in Slack allowing you to see the gif before it is sent and change it if needed.

Thanks,

**Comment by manuelbachl** [comment]
Hi there, i think this question is more related to RC itself. I'm not sure whether RC supports previews. In RC does, there shouldn't be Any Problems to realize this behavior. I'll try to get some info on this

**Comment by muhv7** [comment]
I'd also love that option. Something like Telegram does would be very sweet!

**Comment by graywolf336** [comment]
Temporary work around would be to send a direct message to the user with a list of links to the images and they can copy paste. Sadly the suggestion piece isn't even planned. :(

**Comment by nicodmf** [comment]
Good idea, this is a very important feature for our users

**Comment by manuelbachl** [comment]
Possibly I'll find some time to have a look at this on coming weekend - currently we're kinda packed with work :-/ If some dev around has a working solution, just feel free to open a pull-request ;-)

**Comment by graywolf336** [comment]
Honestly, I wouldn't bother adding this for the integration and would just focus on the App piece since that has support for it already. :)

**Comment by manuelbachl** [comment]
Yes @graywolf336 this were my thoughts aswell. The app-thing will take this week I think, so updates incoming on this issue

**Comment by jwallet** [comment]
looks like they merged a reply preview template in march for whatsapp, i will look into their code

**Comment by jnanes** [comment]
Would love to see this feature added.

**Comment by timlem** [comment]
+1 for this feature added. Looks like everyone is familiar with it but just for reference here's a screencast: [http://recordit.co/KNFRZ1OgiO]

**Comment by madmus** [comment]
> At least give us the ability to send a gif by id ie: !gif id:11ahZZugJHrdLO this should be simple to implement

this should be the easiest implementation 5min work at top.
